### PR TITLE
FUH-CH

### PR DESCRIPTION
**Pull request by abelzer212**
## Purpose

Add FUH-CH Measure

#### Linked Issues to Close

https://qmacbis.atlassian.net/browse/OY2-8942

#### Pull Request Creator Checklist

- [X] This PR has an associated issue or issues.
- [X] The associated issue(s) are linked above.
- [X] This PR meets all acceptance criteria for those issues.
- [X] This PR and linked issue(s) are adequately documented
- [X] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.
- [X] Someone has been assigned this PR.
- [X] At least one person has been marked as reviewer on this PR.

#### Pull Request Reviewer/Assignee Checklist

- [ ] This PR has an associated issue or issues.
- [ ] The associated issue(s) are linked above.
- [ ] This PR meets all acceptance criteria for those issues.
- [ ] This PR and linked issue(s) are adequately documented
- [ ] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.

### CODE REVIEW

File: services/ui-src/src/measures/2021/FUHCH/index.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  }, [setValidationFunctions]);
+
+  const performanceMeasureArray = getPerfMeasureRateArray(data, PMD.data);
+
+  // Conditional check to let rate be readonly when administrative data is the only option or no option is selected
+  const dataSourceWatch = useWatch({ name: "DataSource" });
+  const rateReadOnly =
+    dataSourceWatch?.every(
+      (source: string) => source === "AdministrativeData"
+    ) ?? true;
```
Comment (daniel-belcher) [inline services/ui-src/src/measures/2021/FUHCH/index.tsx:36]: This functionality is built into the CMQ Performance Measure as a default.

File: services/ui-src/src/measures/2021/index.tsx [hunk 2]
```diff
@@ -22,21 +22,22 @@ import { PQI05AD } from "./PQI05AD";
...
   "NCIDDS-AD": NCIDDSAD,
+  "OUD-AD": OUDAD,
   "PDENT-CH": PDENTCH,
-  "CCW-AD": CCWAD,
-  "PQI08-AD": PQI08AD,
   "PQI01-AD": PQI01AD,
-  "PQI15-AD": PQI15AD,
   "PQI05-AD": PQI05AD,
+  "PQI08-AD": PQI08AD,
+  "PQI15-AD": PQI15AD,
```
Comment (daniel-belcher) [inline services/ui-src/src/measures/2021/index.tsx:40]: Like that you re-alphabetized this list.

**Review by daniel-belcher — DISMISSED** [review]

**Review by jphechter — DISMISSED** [review]
This looks good!

File: services/ui-src/src/measures/2021/FUHCH/validation.ts [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  validateNumeratorsLessThanDenominators,
+  validateEqualDenominators,
+  validateNoNonZeroNumOrDenom,
+  validateReasonForNotReporting,
+  validateRequiredRadioButtonForCombinedRates,
+} from "../../globalValidations/validationsLib";
+import { getPerfMeasureRateArray } from "../../globalValidations";
+import { FormData } from "./types";
+
+const validate7DaysGreaterThan30Days = (data: any) => {
```
Comment (charmcitygavin) [inline services/ui-src/src/measures/2021/FUHCH/validation.ts:14]: Could be this a reusable helper function that other measures could use? For example, FUMAD also has a `validate7DaysGreaterThan30Days` function.

File: services/ui-src/src/measures/2021/IETAD/index.tsx [hunk 4]
```diff
@@ -28,13 +28,6 @@ export const IETAD = ({
 
   const performanceMeasureArray = getPerfMeasureRateArray(data, PMD.data);
 
-  // Conditional check to let rate be readonly when administrative data is the only option or no option is selected
-  const dataSourceWatch = useWatch({ name: "DataSource" });
-  const rateReadOnly =
-    dataSourceWatch?.every(
-      (source: string) => source === "AdministrativeData"
-    ) ?? true;
-
```
Comment (daniel-belcher) [inline services/ui-src/src/measures/2021/IETAD/index.tsx:37]: Thank you for removing this from the go-to copy measure.

**Review by daniel-belcher — APPROVED** [review]

File: services/ui-src/src/measures/2021/FUHCH/index.tsx [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+        measureAbbreviation={measureId}
+      />
+
+      {!isNotReportingData && (
+        <>
+          <CMQ.StatusOfData />
+          <CMQ.MeasurementSpecification type="HEDIS" />
+          <CMQ.DataSource />
+          <CMQ.DateRange type="child" />
+          <CMQ.DefinitionOfPopulation />
```
Comment (jdinh8124) [inline services/ui-src/src/measures/2021/FUHCH/index.tsx:45]: You'll still need the childMeasure prop here

File: tests/cypress/support/commands.js [hunk 6]
```diff
@@ -19,8 +19,11 @@ Cypress.Commands.add("goToAdultMeasures", () => {
   cy.get('[data-cy="ACS"]').click();
 });
 
-// Visit Child Core Measure: Medicaid & CHIP
-Cypress.Commands.add("goToChildCoreSetMeasures", () => {
+// Visit Child Core Set Measures
+Cypress.Commands.add("goToChildMeasures", () => {
+  cy.get('[data-cy="Add Child Core Set"]').click();
+  cy.get('[data-cy="ChildCoreSet-ReportType1"]').click();
+  cy.get('[data-cy="Create"]').click();
```
Comment (benjaminpaige) [inline tests/cypress/support/commands.js:26]: i think we are using goToChildCoreSetMeasures somewhere, can we keep the add child coreset functionality seperate? 

ex. in CIS-CH:
```
cy.deleteChildCoreSets();
cy.addCombinedChildCoreset();
cy.goToChildCoreSetMeasures();
cy.goToMeasure("CIS-CH");
```

File: services/ui-src/src/measures/2021/FUHCH/validation.ts (hunk 3 above)
Comment (jphechter) [reply services/ui-src/src/measures/2021/FUHCH/validation.ts:27]: Not a bad idea. Maybe in the future we could make this a generic function that just does an "is greater than" comparison against 2 specified values. I think this should be similar to the kind of info you'd want from `validateDenominatorGreaterThanNumerator`.

**Review by benjaminpaige — APPROVED** [review]

**Pull request merged**